modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -13,103 +16,118 @@ Contact your customer service representative about pricing for this feature.
 
 image::load-balancing.png[load_balancing]
 
-== About High-Reliability Applications
+== High-Reliability Applications
 
-A *high-reliability* application must meet the following requirements:
+A high-reliability application must meet the following requirements:
 
 . Zero tolerance for message loss
 . A reliable underlying enterprise service bus (Mule)
 . Highly reliable individual connections
 
-The feature known as *transactionality* tracks application event sequences to ensure that each message-processing step gets completed successfully, and therefore, no messages get lost or processed incorrectly. If a step fails, for some reason, the transactional mechanism rolls back all previous processing events, then restarts the message processing sequence again.
+The feature known as transactionality tracks application event sequences to ensure that each message-processing step completes successfully, and therefore, no messages get lost or processed incorrectly. If a step fails for some reason, the transactional mechanism rolls back all previous processing events, then restarts the message processing sequence again.
 
-Transports such as JMS, VM, and JDBC provide built-in transactional support, thus ensuring that messages get processed reliably. For example, you can configure a transaction on an inbound JMS connection endpoint to remove messages from the JMS server only after the transaction has been committed. This ensures that the original message remains available for reprocessing if an error occurs during the processing flow.
+Connectors such as JMS, VM, and JDBC provide built-in transactional support, thus ensuring that messages get processed reliably. For example, you can configure a transaction on an inbound JMS connection endpoint to remove messages from the JMS server only after the transaction has been committed. This practice ensures that the original message remains available for reprocessing if an error occurs during the processing flow.
 
-You must use *XA transactions* to move messages between dissimilar transports that support transactions. This ensures that the Mule runtime commits associated transactions from all the dissimilar transports as a single unit. See xref:transaction-management.adoc[Transaction Management] for more information on XA and other types of transactions supported by Mule runtimes.
+You must use XA transactions to move messages between dissimilar Connectors that support transactions. This ensures that the Mule runtime engine commits associated transactions from all the dissimilar connectors as a single unit.
```

Comment:
Capitalize connectors consistently - either initial capped or not. Check the doc to see how it is usually capitalized,